src/components/pages/NotFound/NotFound.module.css

```diff
@@ -0,0 +1,52 @@
+/* NotFound container */
+.NotFound_container {
+  display: grid;
+  place-items: center;
+  height: 100vh;
+  background-color: var(--deep-jungle-blue);
+  color: var(--primary-white);
+  position: relative;
+  z-index: -2;
```

Comment:
I think we don't need z-index in either of the containers.

Reply:
just in case , to avoid this ↓ when the width is small  
![image](https://github.com/user-attachments/assets/cc0068bb-d0d2-4ed5-a547-dc2a9e916a36)